ironfish/src/primitives/rawTransaction.ts

```diff
@@ -197,6 +202,14 @@ export class RawTransactionSerde {
       bw.writeVarString(mint.name, 'utf8')
       bw.writeVarString(mint.metadata, 'utf8')
       bw.writeBigU64(mint.value)
+      if (TransactionFeatures.hasMintTransferOwnershipTo(raw.version)) {
```

Comment:
I would throw an error if `transferOwnershipTo` is set but not supported

Reply:
Added: https://github.com/iron-fish/ironfish/pull/4234/commits/bb31f2c3fe33388604fe622fa98ac3b3b1938ca7